ci-operator/step-registry/sandboxed-containers-operator/get-kata-rpm/sandboxed-containers-operator-get-kata-rpm-ref.yaml

```diff
@@ -17,5 +17,7 @@ ref:
     default: "3.13.0-1.rhaos4.18.el9"
   - name: KATA_RPM_BUILD_MD5SUM
     default: "da576553a2d5925b67ddaf0591454e48"
+  - name: KATA_RPM_BUILD_TASK
+    default: ""
```

Comment:
Based on the value from the next patch `2818/67952818` I think it'd be worth documenting the format as it doesn't really look like a plain BUILD_TASK. Could you please elaborate and provide a sample so people find it more easily? (even in 2 months, or in a year)

Reply:
Good point. I know that `67952818` is actually the real task ID, but I haven't found the meaning of `2818` so I called it all "ID" for the lack of a better name. I will revisit that.

Reply:
Notice that it uses the last four digits of the task ID plus the ID itself to make the URL. So I changed the code so that now users need to pass only the task ID.